templates/navigation-bar.twig

```diff
@@ -0,0 +1,131 @@
+                                                        {% else %}
+                                                            <option value="{{ key }}">{{ website }}</option>
+                                                        {% endif %}
+                                                    {% endfor %}
+                                                </select>
+                                            </div>
+                                        </li>
+                                        <li class="nav-item active hidden-md-down hidden-lg-down">
+                                            <a href="#">ACT</a>
+                                        </li>
+                                        <li class="nav-item hidden-md-down hidden-lg-down">
+                                            <a href="#">EXPLORE</a>
+                                        </li>
+                                        <li class="nav-item hidden-md-down hidden-lg-down">
+                                            <a href="#" class="btn btn-secondary">DONATE</a>
+                                        </li>
+                                        <li class="nav-item loginSignUpWrap">
+                                            <i class="fa fa-user-circle-o"></i>
+                                            <a href="#">Log in</a> <span> / </span>
+                                            <a href="#">Sign up</a>
+                                        </li>
+                                        <li class="nav-item nav-search-wrap">
+                                            <form class="form">
+                                                <input class="form-control" type="text" placeholder="Search" aria-label="Search">
+                                                <button class="top-nav-search-btn" type="submit"><i class="fa fa-search"></i></button>
+                                            </form>
+                                        </li>
+                                    </ul>
+                                </div>
+                            </nav>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="clearfix"></div>
+        <div class="sub-navigation-md hidden-md-down hidden-xl-up">
+            <div class="container">
+                <ul class="list-unstyled clearfix">
+                    <li class="nav-item">
+                        <a href="#">ACT</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#">EXPLORE</a>
+                    </li>
+                    <li class="nav-item text-right donate-btn">
+                        <a href="#" class="btn btn-secondary">DONATE</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    {% endblock %}
+{% endautoescape %}
```

Comment:
Needs a new line at the end of the file.

Reply:
Some text seems to be registered for translation and some isn't. Is this intentional?

Reply:
Hi Dasha, almost at the same time I pushed a new commit with the rest of the text registered for translations :)